AEPIdentity/Tests/TestHelpers/TestableExtensionRuntime.swift

```diff
@@ -53,6 +53,10 @@ class TestableExtensionRuntime: ExtensionRuntime {
     }
 
     func getSharedState(extensionName: String, event: Event?, barrier: Bool) -> SharedStateResult? {
+        if event == nil {
```

Comment:
We may need to make some other changes to the TestableRuntimes to ensure that it mimicks the EventHub behavior by fetching latest when a `nil` event is passed in.